js/build_map.js

```diff
@@ -100,6 +100,9 @@ function create_map() {
   // add legend
   addLegend(minWateruse, maxWateruse);
   
+  // add watermark
+  addUSGS();
```

Comment:
can we just change to `addUSGSWatermark` or `addWatermark`? Might add something else USGS-y and then this will be confusing?

Reply:
sure